### DISCUSSION TITLE
Have I Been Pwned

### DISCUSSION THREAD

**Discussion by 1RandomDev** [dani-garcia/vaultwarden#1814] [Ideas]
I just noticed that you can make inofficial "api" requests to https://haveibeenpwned.com/unifiedsearch/{email} to get information from HIBP without buying a key. Couldn't that be used for vaultwarden?

**Comment by BlackDex** [comment]
That would be an abuse i of there service in my opinion.

**Reply by 1RandomDev** [reply]
Ok true, but I think it would be easy to add in a fork. I've never bought a key, but the response looks very similar to the one of the official api I think

**Comment by BlackDex** [comment]
Besides that, i did tested it a year ago, it works, but i don't want that service to be misused. If someone wants it integrated they should buy a key.

**Reply by 1RandomDev** [reply]
Ok for me it's not worth buying a key for one private user

**Reply by BlackDex** [reply]
That is why i created a simple on click link, just one step away
